src/main/features/offer/routes/declaration.ts

```diff
@@ -23,15 +24,15 @@ function renderView (form: Form<Declaration>, res: express.Response) {
 /* tslint:disable:no-default-export */
 export default express.Router()
   .get(
-    Paths.declarationPage.uri,
+    Paths.declarationPage.uri, OfferAcceptedGuard.requestHandler,
```

Comment:
Because declaration is used for both singing an offer and counter signing I think current implementation would break counter signing.

@ellie-harrison can you make sure that counter signing works?

If not we would need to change the guard to mimic the following logic I found in the `declaration.ts`:

```
        if (user.id === claim.defendantId) {
          await OfferClient.countersignOffer(claim.externalId, user)

          res.redirect(Paths.settledPage.evaluateUri({ externalId: claim.externalId }))
        } else {
          await OfferClient.acceptOffer(claim.externalId, user)

          res.redirect(Paths.acceptedPage.evaluateUri({ externalId: claim.externalId }))
        }
```

Reply:
@ellie-harrison any update on this?

Reply:
This does affect countersigning, currently implementing a solution to stop that happening.